src/test/resources/org/broadinstitute/hellbender/engine/reads_data_source_test4.xor.bam.bai

```diff
@@ -0,0 +1 @@
+QTPPPRPPPBPPQPPP�PPPPPPPSPPPPPP�PPRPPP�PPPPPPPSPPPPPPTPPPPPPPPPPPPPPPQPPP�PPPPPPPRPPPBPPQPPPSPPPPPP�UPPPPPP�PPRPPPSPPPPPP�UPPPPPPTPPPPPPPPPPPPPPPQPPPSPPPPPPRPPPBPPQPPP�UPPPPPPPP�QPPPP�PPRPPP�UPPPPPPPP�QPPPPSPPPPPPPPPPPPPPPQPPP�UPPPPPPPPPPPPPPPPPPPPPP
```

Comment:
We'd have to write them to a bucket. This will create problems if more than one person/tool is running the test simultaneously.